app/src/main/res/layout/activity_request_detail.xml

```diff
@@ -24,7 +24,7 @@
         app:layout_constraintHorizontal_bias="0.957"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent"
-        tools:text="Lola Zink" />
+        tools:text="@string/lola_zink" />
```

Comment:
This change is not required. The Strings that are used in the `tools:text` attribute are only for preview purposes.

Reply:
Noted.
All the strings in the tools: text will be removed

Reply:
> This change is not required. The Strings that are used in the `tools:text` attribute are only for preview purposes.

The required task is done